viewer/packages/360-images/src/annotation/ImageAnnotationObject.ts

```diff
@@ -49,13 +49,22 @@ export class ImageAnnotationObject implements Image360Annotation {
   private static createObjectData(detection: AnnotationData): ImageAnnotationObjectData | undefined {
     if (isAnnotationsObjectDetection(detection)) {
       if (detection.polygon !== undefined) {
-        return new PolygonAnnotationData(detection);
+        return new PolygonAnnotationData(detection.polygon);
```

Comment:
Why we decided to pass the polygon object here? Is it because of the type in `PolygonAnnotationData`?

Reply:
Yes, I changed the constructor for `PolygonAnnotationData` to take the polygon directly, instead of the asset data object. This way, it can be used with different asset data types, like AssetLink and ObjectDetection.